components/SkeletonCard.vue

```diff
@@ -0,0 +1,28 @@
+<template>
+  <div class="skeleton-card">
+    <v-skeleton-loader type="image" />
+    <v-skeleton-loader type="heading" class="mt-1" />
+  </div>
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+
+export default Vue.extend({});
+</script>
+
+<style>
```

Comment:
Same for this, can we scope it?

Reply:
Scoping this breaks things because I'm targeting classes in the v-skeleton-loader

Reply:
I hadn't seen that before. I'll add it now.